Package.swift

```diff
@@ -42,10 +45,6 @@ let package = Package(
 for target in package.targets {
     var settings = target.swiftSettings ?? []
     settings.append(.enableExperimentalFeature("StrictConcurrency=complete"))
```

Comment:
Can you add all the various upcoming feature flags, especially NonisolatedNonsendingByDefault and InternalImportsByDefault. Probably add the union of https://github.com/apple/swift-configuration/blob/b1fc0c025c10db208c22291ec46310f2d60ce446/Package.swift#L168 and https://github.com/apple/swift-temporal-sdk/blob/7c7e1a3e961136d4d12d201649cc6b144fcf4ee6/Package.swift#L250